btr-web/btr-common-components/stores/account.ts

```diff
@@ -0,0 +1,135 @@
+import Axios from 'axios'
+import { StatusCodes } from 'http-status-codes'
+import { defineStore } from 'pinia'
+import { AccountI } from '~/interfaces/account-i'
+import { ErrorCategoryE } from '~/enums/error-category-e'
+import { ErrorI } from '~/interfaces/error-i'
+import { KCUserI } from '~/interfaces/kc-user-i'
+import { addAxiosInterceptors } from '~/utils/axios'
+
+/** Manages bcros account data */
+export const useBcrosAccount = defineStore('bcros/account', () => {
+  // keycloak info
+  const keycloak = useBcrosKeycloak()
+  // selected user account
+  const currentAccount: Ref<AccountI> = ref({} as AccountI)
+  const currentAccountName = computed((): string => currentAccount.value?.label || '')
+  // user info
+  const user = computed(() => keycloak.kcUser)
+  const userAccounts: Ref<AccountI[]> = ref([])
+  const userFirstName: Ref<string> = ref(user.value?.firstName || '-')
+  const userLastName: Ref<string> = ref(user.value?.lastName || '')
+  const userFullName = computed(() => `${userFirstName.value} ${userLastName.value}`)
+  // errors
+  const errors: Ref<ErrorI[]> = ref([])
+  // api request variables
+  const axios = addAxiosInterceptors(Axios.create())
+  const apiURL = useRuntimeConfig().public.authApiURL
+
+  /** Get user information from AUTH */
+  async function getAuthUserProfile (identifier: string) {
+    return await axios.get<KCUserI | void>(`${apiURL}/users/${identifier}`)
```

Comment:
currently we only make this call once for the first name / last name of bceid users since that information isn't updated in keycloak. I save that data where this is called